src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -125,17 +125,20 @@ public static void assertCommandSuccess(Command command, Model actualModel, Stri
      * Executes the given {@code command}, confirms that <br>
      * - a {@code CommandException} is thrown <br>
      * - the CommandException message matches {@code expectedMessage} <br>
-     * - the address book, filtered member list and selected member in {@code actualModel} remain unchanged
+     * - the address book, filtered member list, filtered event list, selected member and selected event
```

Comment:
Do we keep this as address book and replace address book with CCACommander in the later PR?